common/src/main/java/com/wynntils/core/functions/arguments/FunctionArguments.java

```diff
@@ -26,38 +25,59 @@ public Argument getArgument(String name) {
         return this.lookupMap.get(name);
     }
 
-    public static final class Builder {
+    // A builder for functions with required arguments
```

Comment:
Here was the answer to my previous question (if it's confusing, it's because I ended up reviewing from bottom to top... don't ask why 🤷 )

Maybe make Builder abstract and have a second concrete subclass RequiredArgumentBuilder, to clarify the difference?

Reply:
Done.